components/subscribe-to-stanford/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://localhost:3000/schemas/v1.json#",
   "name": "subscribe-to-stanford-report",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
I incremented the version of subscribe-to-stanford-report here - LMK if that's ok